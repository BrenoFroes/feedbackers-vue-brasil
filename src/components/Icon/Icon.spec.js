import { shallowMount } from '@vue/test-utils'
import Icon from '.'
import Copy from './Copy'
import { routes } from '../../router'
import { createRouter, createWebHistory } from 'vue-router'

const router = createRouter({
  history: createWebHistory('/'),
  routes
})

describe('<Icon/>', () => {
  it('should render a Icon', async () => {
    router.push('/')
    await router.isReady()
    const wrapperCopy = shallowMount(Copy)
    const wrapper = shallowMount(Icon, {
      global: {
        plugins: [router]
      },
      propsData: {
        name: wrapperCopy
      }
    })
    expect(wrapper.html()).toMatchSnapshot()
    expect(wrapper.vm).toBeDefined()
  })
})
