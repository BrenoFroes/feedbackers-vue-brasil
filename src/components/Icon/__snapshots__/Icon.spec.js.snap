// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Icon/> should render a Icon 1`] = `<anonymous-stub name="[object Object]"></anonymous-stub>`;
